app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ForgotPasswordsController < ApplicationController
+
+  def create
+    user = User.find_by_email(params[:email])
+    if user
+      AppMailer.send_forgot_password(user).deliver
+      redirect_to forgot_password_confirmation_path
+    else
+      flash[:error] = params[:email].blank? ? "Email cannot be blank." : "There is no user with that email address."
```

Comment:
We probably do not want to do this, because of security reasons. Almost always we want to set a positive flash message to user about sending him details. The reason is, it is good practice to not give out much information about what we have in database to a malicious user. Normal user would just retry in 5 minutes or the flash would show up targeted email address. Obviously our application would not send mail to non-existing user/email.